modules/nw-cfg-config-all-multi-cni.adoc

```diff
@@ -0,0 +1,167 @@
+// Module included in the following assemblies:
+//
+// * networking/setting-interface-level-network-sysctls.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="nw-enabling-all-multi-cni_{context}"]
+= Enabling all-multicast mode by using the tuning CNI
+
+You can enable all-multicast mode by using the tuning Container Network Interface (CNI) meta plugin.
+
+The following procedure describes how to configure the tuning CNI to enable the all-multicast mode.
+
+.Procedure
+
+. Create a network attachment definition, such as `tuning-example.yaml`, with the following content:
++
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: <name> <1>
+  namespace: default <2>
+spec:
+  config: '{
+    "cniVersion": "0.4.0", <3>
+    "name": "<name>", <4>
+    "plugins": [{
+       "type": "<main_CNI_plugin>" <5>
+      },
+      {
+       "type": "tuning", <6>
+       "allmulti": true <7>
+        }
+      }
+     ]
+}
+----
+<1> Specifies the name for the additional network attachment to create. The name must be unique within the specified namespace.
+<2> Specifies the namespace that the object is associated with.
+<3> Specifies the CNI specification version.
+<4> Specifies the name for the configuration. Match the configuration name to the name value of the network attachment definition.
+<5> Specifies the name of the main CNI plugin to configure.
+<6> Specifies the name of the CNI meta plugin.
+<7> Change the all-multicast mode of interface. If enabled, all multicast packets on the network will be received by the interface.
++
+An example YAML file is shown here:
++
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: setallmulti
+  namespace: default
+spec:
+  config: '{
+    "cniVersion": "0.4.0",
+    "name": "setallmulti",
+    "plugins": [
+      {
+        "type": "bridge"
+      },
+      {
+        "type": "tuning",
+        "allmulti": true
+      }
+    ]
+  }'
+----
```

Comment:
With the changes you made to the wording of step 1 with the sample YAML, I don't understand why this identical example needs to be repeated here. What am I missing? A user can copy paste the previous YAML code and the callouts will not be included, or at least that's my understanding. I'd say that this example section can probably just be removed as redundant. WDYT?

Reply:
i want to leave this in as it is part of a end to end example a user could follow on their own test system so I think it is valid to leave in as a populated example of what this file should look like.